src/Controller/SsoController.php

```diff
@@ -17,31 +26,23 @@ public function login(Request $request)
         $password = $request->get('password');
 
         if (!$username || !$password) {
-            $response->setStatusCode(401);
-            $response->setContent('Username or password not provided');
-            return $response;
+            return $this->error($response,'Username or password not provided');
         }
 
         try {
             $user = $this->getDoctrine()->getRepository(User::class)->findByUsernameOrEmail($username);
         } catch (NoResultException $e) {
-            $response->setStatusCode(401);
-            $response->setContent('Username does not exist');
-            return $response;
+            return $this->error($response,'Username does not exist');
         }
 
         $validPassword = $this->get('security.password_encoder')->isPasswordValid($user, $password);
         if (!$validPassword) {
-            $response->setStatusCode(401);
-            $response->setContent('Wrong password');
-            return $response;
+            return $this->error($response,'Wrong password');
         }
 
         $activeInTeam = count($user->getActiveMemberships()) > 0;
         if (!$activeInTeam) {
-            $response->setStatusCode(401);
-            $response->setContent('User does not have any active team memberships');
-            return $response;
+            return $this->error($response,'User does not have any active team memberships');
```

Comment:
It seems that the status code `401` is missing from the parameters passed to the `error function`.
```suggestion
            return $this->error($response,'User does not have any active team memberships', 401);
```